index.js

```diff
@@ -8,14 +8,15 @@ var gutil         = require('gulp-util')
   , temp          = require('temp')
   , spawn         = require('cross-spawn')
   , Q             = require('q')
+  , elm_make_dev  = 'node_modules/.bin/elm-make'
   , elm_make      = 'elm-make'
   , defaultArgs   = ['--yes']
   , PLUGIN        = 'gulp-elm';
 
 function processMakeOptions(options, output) {
   var args   = defaultArgs
     , ext    = '.js'
-    , exe    = elm_make;
+    , exe    = fs.existsSync(elm_make_dev) && fs.statSync(elm_make_dev).isFile() ? elm_make_dev : elm_make;

```

Comment:
Yes, its deprecated. I dared to use it because in the code there is a `fs.exists` anyway. I can take a look at it and replace both `exists` and `existsSync`.